country-info.csv

```diff
@@ -152,3 +152,4 @@ Zhejiang University,asia
 Bielefeld University,europe
 KTH Royal Institute of Technology,europe
 UFRGS,southamerica
+Universidade Federal de Uberlândia,southamerica
```

Comment:
For symmetry with Rio Grande do Sul, please abbreviate to UFU. Thanks.

Reply:
Thanks for the review.
The Brazilian universities: Universidade Federal de Minas Gerais, University of São Paulo, University of Campinas are not abbreviated. Just UFRGS is. Should I abbreviate to UFU?

Reply:
Done.

Reply:
Some (but not all) of these names are really long and they mess up the UI. Thanks.

Reply:
I'll be glad to abbreviate the rest.

Reply:
That'd be great, thanks.

Reply:
(separate PR)

Reply:
Sure. Separate PR per university?

Reply:
One separate PR for all universities is fine. Thanks!